### PR TITLE
Librsvg

### PR DESCRIPTION
**Pull request by joebonrichie**

### CODE REVIEW

File: l/librsvg/stone.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - pkgconfig(gdk-pixbuf-2.0)
+    - pkgconfig(gobject-introspection-1.0)
+    - pkgconfig(libpng)
+    - pkgconfig(libxml-2.0)
+    - pkgconfig(pango)
+    - pkgconfig(xdamage)
+    - pkgconfig(xshmfence)
+environment : |
+    export HOME=$(pwd)
+    export CARGO_HTTP_CAINFO=/usr/share/defaults/etc/ssl/certs/ca-certificates.crt
```
Comment (ermo) [inline l/librsvg/stone.yml:33]: This should probably be using the `%(vendordir)` prefix instead?

**Review by ermo — CHANGES_REQUESTED** [review]
Small nit, otherwise looks good to go?

**Comment by der-eismann** [comment]
Again, monitoring.yaml please :) and it would be great if you could rebuild app stream with SVG support as well, was just merged

**Review by ikeycode — APPROVED** [review]
danke!

**Pull request merged**